src/Acuminator/Acuminator.Utilities/Roslyn/Semantic/ITypeSymbolExtensions.cs

```diff
@@ -331,27 +341,29 @@ private static string GetClrStyleTypeFullNameForNotNestedType(this ITypeSymbol n
             return initializers;
         }
 
-        public static ImmutableArray<StaticConstructorInfo> GetStaticConstructors
-            (this INamedTypeSymbol typeSymbol, CancellationToken cancellation = default)
-        {
-            typeSymbol.ThrowOnNull(nameof(typeSymbol));
+		public static ImmutableArray<StaticConstructorInfo> GetStaticConstructors
```

Comment:
tabs/spaces mix

Reply:
The VS shows no mixed tabs. I think the method was created with this indentation. I'll fix it

Reply:
fixed